flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/ExecutionContext.java

```diff
@@ -232,6 +232,15 @@ public SessionState getSessionState() {
 		}
 	}
 
+	/**
+	 * Executes the given Runnable using the execution context's classloader as thread classloader.
+	 */
+	private void wrapClassLoader(Runnable runnable) {
```

Comment:
Since we have this method now, could you also correct the usage of old `wrapClassLoader(Supplier<R> supplier)` and `return null` inside? I think they should be changed to use this one.